src/group/managed_group/mod.rs

```diff
@@ -1,7 +1,6 @@
-use crate::ciphersuite::*;
 use crate::codec::*;
```

Comment:
Why is this removed?

Reply:
Because it's one of the many warnings in here ...